packages/mgt-chat/src/statefulClient/StatefulGraphChatClient.ts

```diff
@@ -405,6 +416,13 @@ class StatefulGraphChatClient implements StatefulClient<GraphChatClient> {
     }
   }
 
+  public setStatus(status: GraphChatClient['status']) {
```

Comment:
Let's refactor the inline type for status out to it's own exported type so that we can use it directly